components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -203,12 +217,22 @@ export const PrebuildDetailPage: FC = () => {
                             </div>
                             <div className="h-112 border-pk-border-base">
                                 <Suspense fallback={<div />}>
-                                    <WorkspaceLogs
-                                        classes="h-full w-full"
-                                        xtermClasses="absolute top-0 left-0 bottom-0 right-0 mx-6 my-0"
-                                        logsEmitter={logEmitter}
-                                        isLoading={isStreamingLogs}
-                                    />
+                                    {logNotFound ? (
+                                        <div className="px-6 py-4 h-full w-full bg-pk-surface-primary text-base flex items-center justify-center">
+                                            <Text className="w-[22rem] text-center">
+                                                Logs of failed prebuilds are inaccessible. Use{" "}
+                                                <code>gp validate --prebuild --headless</code> in a workspace to see
```

Comment:
We could add link to document https://www.gitpod.io/docs/configure/workspaces#validate-your-gitpod-configuration

Reply:
Something worth adds to document

- `--headless` is a hidden flag, maybe make it visible for all? cc @akosyakov I'm not sure why we hide it before
- `--headless` can only pipe one task's log (tested in dogfood, not check code)
- `--headless` may pipe nothing if tasks exit too fast
<img width="1309" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/b2f4edde-00c2-420d-8023-e03d9d40f4e9">

Reply:
Or we recommend to use `--prebuild` only then let users go to visit debug ws link and find logs themselves. (even though prebuild should be headless)

Reply:
I think we should change --prebuild to be headless on by default, and then see whether we really need to document.

Reply:
I pushed a commit linking to the relevant docs. Let's follow this up with updates to `gp validate`?